cli/cmd/plugin/ui/web/tanzu-ui/src/views/management-cluster/aws/aws-mc-common/management-credential-step/ManagementCredentials.tsx

```diff
@@ -153,6 +170,61 @@ function ManagementCredentials(props: Partial<StepProps>) {
         }
     };
 
+    useEffect(() => {
+        if (awsState[STORE_SECTION_FORM].REGION) {
+            retrieveOsImages(awsState[STORE_SECTION_FORM].REGION);
+        }
+    }, [awsState[STORE_SECTION_FORM].REGION]);
+
+    function retrieveOsImages(region: string) {
+        try {
+            setOsImages([]);
+            if (region !== '') {
+                AwsService.getAwsosImages(region).then((data) => {
+                    setOsImages(data);
+                    setErrorMessage((errorMessage) => {
+                        const copy = { ...errorMessage };
+                        delete copy['OS_IMAGE'];
+                        return copy;
+                    });
+                });
+            }
+        } catch (e) {
+            setErrorMessage({
+                ...errorMessage,
+                OS_IMAGE: e,
+            });
+        }
+    }
+
+    function showErrorInfo() {
+        if (connectionStatus === CONNECTION_STATUS.CONNECTED && JSON.stringify(errorMessage) !== '{}') {
+            return (
+                <div>
+                    <div className="error-text">Error Occurs</div>
+                    <br />
+                    {Object.keys(errorMessage).map((errorField) => {
+                        return (
+                            <CdsControlMessage status="error" key={errorField}>
+                                {errorMessage[errorField]}
+                            </CdsControlMessage>
+                        );
+                    })}
+                    <br />
+                </div>
+            );
+        }
+        return;
+    }
+
+    const canContinue = (): boolean => {
+        return (
+            connectionStatus === CONNECTION_STATUS.CONNECTED &&
+            getValues('EC2_KEY_PAIR') !== undefined &&
```

Comment:
Should be able to use AWS FIELDS.EC2_KEY_PAIR enum (I might have naming a little wrong)

Reply:
Yes sir. Changed to 'AWS_FIELDS.EC2_KEY_PAIR'